funnel/models/user.py

```diff
@@ -809,8 +809,12 @@ def __init__(self, representation: str) -> None:
         self.fullname = self.title = self.pickername = representation
 
     def __str__(self) -> str:
-        """Represent user account as a string."""
-        return self.pickername
+        """
+        Represent user account as a string.
+
+        `str()` because `__str__()` needs to return a string object.
```

Comment:
When is `pickername` not a str? Also, this should be comment, not docstring.

Reply:
By default it's `LazyString` because we create model instance like this `deleted_user = DuckTypeUser(__("[deleted]"))`.